projects/angular-ngrx-material-starter/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "src",
+    "paths": {
+      "@root/*": ["../../../*"],
+      "@app/*": ["app/*"],
+      "@core/*": ["app/core/*"],
+      "@shared/*": ["app/shared/*"],
+      "@assets/*": ["assets/*"],
+      "@environments/*": ["environments/*"]
```

Comment:
It was like this before but decided to move away from it cuz it breaks in stuff like `stackblitz` etc... So this will not be added

Reply:
Cool I'll revert that commit